src/Pivot.vue

```diff
@@ -360,6 +382,7 @@ $carret-bold-svg: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/200
 }
 
 .pivottable {
+  margin: 0 15px;
```

Comment:
Same as `p-0` above.
As is: padding-x: 15px; margin: 0;
To be: margin-x: 15px; padding; 0;